### DISCUSSION TITLE
Nautobot 2.0 Release Information

### DISCUSSION THREAD

**Discussion by mattmiller87** [nautobot/nautobot#3540] [General]
Does a list of pre-canned responses to common questions Nautobot administrators may have on the changes coming for Nautobot 2.0 exist? Perhaps a short document or slide deck with links pointing in places for more details.

**Comment by bryanculver** [comment]
@mattmiller87 You can start by taking a look at: https://docs.nautobot.com/projects/core/en/next/installation/upgrading-from-nautobot-v1/ but it isn't expressly a FAQ, nor is it deeply technical for a migration guide for developers. We are not near enough to the release yet to have such information available. Once we start entering the beta phases of 2.0 will we be able to write documentation that may have significant changes at this point.

**Reply by bryanculver** [reply]
Our docs have been updated:

* https://docs.nautobot.com/projects/core/en/next/user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1/
* https://docs.nautobot.com/projects/core/en/next/development/jobs/migration/from-v1/
* https://docs.nautobot.com/projects/core/en/next/development/apps/migration/from-v1/
